x-pack/plugins/ml/common/types/file_datavisualizer.ts

```diff
@@ -84,7 +84,9 @@ export interface Settings {
 }
 
 export interface Mappings {
-  [key: string]: any;
+  properties: {
```

Comment:
the `_meta` property is needed here now.

```
export interface Mappings {
  _meta: {
    created_by: string;
  };
  properties: {
    [key: string]: any;
  };
}
```

Reply:
Should it be `_meta?: {` because it won't be present in the initial response?  Or is this only used to validate the mappings the UI creates based on the backend response?

Reply:
it's only used for type checking the mappings object structure in the code, not at run time.
But if it is an optional property, it would be best that the type is correct, so yes `_meta?:`